README.md

```diff
@@ -61,8 +61,7 @@ we support!
 
 ## :books: Documentation
 
-- See our documentation site
-  [here](https://www.carbondesignsystem.com/get-started/develop/react) for full
+- See our documentation site [here](https://www.carbondesignsystem.com) for full
```

Comment:
```suggestion
- See our documentation site [here](https://www.carbondesignsystem.com/developing/frameworks/react/) for full
```

I bet we can use this link to mirror the get started one that was originally there 👀

Reply:
Oh yeah, will change it accordingly. Thanks!